src/script/conversation/ConversationRepository.ts

```diff
@@ -3543,6 +3543,15 @@ export class ConversationRepository {
       });
     }
 
+    const is1to1Conversation = conversationEntity.is1to1() || conversationEntity.isRequest();
+
+    if (is1to1Conversation) {
+      const otherUserId = conversationEntity.participating_user_ids()[0];
```

Comment:
Maybe add chaining operator to check if users is exist - for more safety. ;)

```suggestion
      const otherUserId = conversationEntity.participating_user_ids()?.[0];
```

Reply:
It's not necessary, `participating_user_ids` is always an array, see https://github.com/wireapp/wire-webapp/blob/runfix/mls-1to1-accept-flicker/src/script/entity/Conversation.ts#L212

Reply:
Ahh yee.. my mistake :D